docs/src/AlgebraicGeometry/standard_constructions.md

```diff
@@ -0,0 +1,22 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["standard_constructions.md"]
+```
+
+# Standard Constructions in Algebraic Geometry
+
+This page documents a collection of standard constructions in Algebraic Geometry
```

Comment:
Shouldn't this be lower case "algebraic geometry" in the text; title case in the headline of course is fine (I am not a native speaker, this is just how I learned it, ignore  me if this is nonsense)
```suggestion
This page documents a collection of standard constructions in algebraic geometry
```